concrete-core-fixture/src/fixture/cleartext_vector_creation.rs

```diff
@@ -0,0 +1,109 @@
+use crate::fixture::Fixture;
+use crate::generation::prototyping::PrototypesCleartextVector;
+use crate::generation::synthesizing::SynthesizesCleartextVector;
+use crate::generation::{IntegerPrecision, Maker};
+use crate::raw::generation::RawUnsignedIntegers;
+use crate::raw::statistical_test::assert_noise_distribution;
+use concrete_commons::dispersion::Variance;
+use concrete_commons::parameters::CleartextCount;
+
+use concrete_core::prelude::{CleartextVectorCreationEngine, CleartextVectorEntity};
+
+/// A fixture for the types implementing the `CleartextVectorCreationEngine` trait.
+pub struct CleartextVectorCreationFixture;
+
+#[derive(Debug)]
+pub struct CleartextVectorCreationParameters {
+    count: CleartextCount,
+}
+
+impl<Precision, Engine, CleartextVector> Fixture<Precision, Engine, (CleartextVector,)>
+    for CleartextVectorCreationFixture
+where
+    Precision: IntegerPrecision,
+    Engine: CleartextVectorCreationEngine<Precision::Raw, CleartextVector>,
+    CleartextVector: CleartextVectorEntity,
+    Maker: SynthesizesCleartextVector<Precision, CleartextVector>,
+{
+    type Parameters = CleartextVectorCreationParameters;
+    type RepetitionPrototypes = ();
+    type SamplePrototypes = (Vec<Precision::Raw>,);
+    type PreExecutionContext = (Vec<Precision::Raw>,);
+    type PostExecutionContext = (CleartextVector,);
+    type Criteria = (Variance,);
+    type Outcome = (Vec<Precision::Raw>, Vec<Precision::Raw>);
+
+    fn generate_parameters_iterator() -> Box<dyn Iterator<Item = Self::Parameters>> {
+        Box::new(
+            vec![CleartextVectorCreationParameters {
+                count: CleartextCount(500),
```

Comment:
Here again, maybe we should just check 1 also.

Reply:
Sure ! Just did it ! Thanks !